### DISCUSSION TITLE
How can folks support the immich developers?

### DISCUSSION THREAD

**Discussion by coltenkrauter** [immich-app/immich#10868] [General]
Is there a [buy me a coffee](https://buymeacoffee.com/) link or similar way that folks could throw some **_$_** support toward immich developers? 

**This project is fantastic** and will increase folks' self-sustainability, ability to ditch the cloud, and cut the cord with big tech photo storage subscriptions. It will save me money.

![image](https://github.com/immich-app/immich/assets/18080897/07a94ba3-ddd9-461c-a990-8bc97890b834)

**Comment by zackpollard** [comment]
We don't accept donations, however we will soon be adding the ability to optionally purchase Immich for a fee. That will be the way to support the project going forwards. This will be totally optional and no features will ever be gated behind this purchase fee.

**Reply by coltenkrauter** [reply]
Wow, that is very cool. Your approach makes me want to support even more. 

Well, I respect not accepting donations at this point. I would also respect linking to developer donation platforms. 

Thanks for all you guys do.
